borg/helpers.py

```diff
@@ -879,7 +929,13 @@ def remove_surrogates(s, errors='replace'):
 def make_path_safe(path):
     """Make path safe by making it relative and local
     """
-    return _safe_re.sub('', path) or '.'
+    if sys.platform != 'win32':
+        return _safe_re.sub('', path) or '.'
+    else:
+        tail = path
+        if path[0:2] == '//' or path[0:2] == '\\\\' or path[1] == ':':
+            drive, tail = os.path.splitdrive(path)
```

Comment:
> if we omit the drive letter, information about where it came from is lost.

That's not entirely wrong, but since paths including either an UNC prefix or a drive letter are absolute, and that does not make sense to store/extract.

Imagine unices had multiple different roots with some arcane syntax, say `host:path`, then including paths from multiple roots in one archive is equally meaningless for overlapping contents as would be including paths from multiple drives in the one archive on Windows.